src/compute/tests/integration_tests.rs

```diff
@@ -163,11 +163,12 @@ async fn test_table_materialize() -> StreamResult<()> {
 
     // Create a `SourceExecutor` to read the changes.
     let source_executor = Executor::new(
-        ExecutorInfo {
-            schema: all_schema.clone(),
-            pk_indices: pk_indices.clone(),
-            identity: format!("SourceExecutor {:X}", 1),
-        },
+        ExecutorInfo::new(
+            all_schema.clone(),
+            pk_indices.clone(),
+            format!("SourceExecutor {:X}", 1),
+            0,
```

Comment:
For this one, the executor ID should be `1`. IIUC, in this integration tests we create more than one executor, so need to assign them different ID.